Tribler/Test/Core/Upgrade/test_torrent_upgrade_63_64.py

```diff
@@ -0,0 +1,118 @@
+import os
+import shutil
+from apsw import Connection
+from nose.tools import raises
+from Tribler.Core.Upgrade.torrent_upgrade64 import TorrentMigrator64
+from Tribler.Test.Core.base_test import TriblerCoreTest
+from Tribler.Test.test_libtorrent_download import TORRENT_FILE
+
+
+class AbstractTorrentUpgrade63to64(TriblerCoreTest):
+
+    FILE_DIR = os.path.abspath(os.path.dirname(os.path.realpath(__file__)))
+    DB_DATA_DIR = os.path.abspath(os.path.join(FILE_DIR, u"../data/upgrade_databases/"))
+
+    def write_data_to_file(self, file_name):
+        file = open(file_name, 'w')
```

Comment:
totally cosmetic remark: you can use with open(...) as file: to make it nicer :)